ChangeLog.md

```diff
@@ -2,6 +2,21 @@
 # DGtal 0.9.1
 
 ## New Features / Critical Changes
+- *Base Package*
+ - Traits class for containers in order to probe their category at compile time.
+   (Jacques-Olivier Lachaud, [#1079](https://github.com/DGtal-team/DGtal/pull/1079))
+ - Generic set operations for arbitrary containers. You may use overloaded operators
+   like &, |, -, ^ on arbitrary containers (list, vector, unordered_set, map, etc).
+   (Jacques-Olivier Lachaud, [#1079](https://github.com/DGtal-team/DGtal/pull/1079))
+
```

Comment:
Can DigitalSet containers could also be considered in these specializations ? It seems to be pretty trivial for the ```DigitalSetByAssociativeContainer``` class for instance.